pages/1.10/installing/oss/cloud/gce/index.md

```diff
@@ -6,283 +6,241 @@ navigationTitle: GCE
+We've provided all the sensible defaults that you would want to play around with DC/OS. Just run this command to deploy a multi-master setup in the cloud. Three agents will be deployed for you. Two private agents, one public agent.
 
-1.  After creating the bootstrap instance, start the instance and run the following from the shell. These commands install prerequisite software on your bootstrap node.
+- There is no git clone of this repo required. Terraform does this for you under the hood.
 
-    ```bash
-    sudo yum update google-cloud-sdk &&
-    sudo yum update &&
-    sudo yum install epel-release &&
-    sudo yum install python-pip &&
-    sudo pip install -U pip &&
-    sudo pip install 'apache-libcloud==1.2.1' &&
-    sudo pip install 'docker-py==1.9.0' &&
-    sudo yum install git ansible
-    ```
+```bash
+terraform init -from-module github.com/dcos/terraform-dcos//gcp
+terraform apply -var gcp_project="your_existing_project"
+```
 
-### Setup RSA public/private keypairs
+### Custom terraform-dcos variables
 
-You must create the RSA public/private keypairs to allow passwordless logins via SSH to the nodes of the DC/OS cluster. This is required by Ansible to create the cluster nodes and install DC/OS on the nodes.
+The default variables are tracked via in the [variables.tf](https://github.com/dcos/terraform-dcos/blob/master/gcp/variables.tf) file. Since this file can be overwritten during updates when you may run `terraform get --update` when you want to fetch new releases of DC/OS to upgrade too, its best to use the [desired_cluster_profile.tfvars](https://github.com/dcos/terraform-dcos/blob/master/gcp/desired_cluster_profile.tfvars.example) and set your custom terraform and DC/OS flags there. This way you can keep track of a single file that you can use manage the lifecycle of your cluster.
 
-**Important:** Replace `ajazam` with your username in these examples.
+For list of supported operating systems for this repo which expands, we're supporting the ones that DC/OS recommends [here](https://docs.mesosphere.com/1.10/installing/oss/custom/system-requirements/). You can find the list that Terraform supports [here](http://github.com/bernadinm/tf_dcos_core).
 
-1.  Run this command to generate the keys:
+To apply the configuration file, you can use this command below.
 
-    ```bash
-    ssh-keygen -t rsa -f ~/.ssh/id_rsa -C ajazam
-    ```
+```bash
+terraform apply -var-file desired_cluster_profile.tfvars
+```
 
-    Do not enter a password when prompted.
+#### Advance YAML Configuration
 
-1.  Make a backup copy of `id_rsa.pub`.
+In this project we have designed to be flexible. Here are the example working variables that allows very deep customization by using a single `tfvars` file.
 
-1.  Open RSA pub key:
+This file can have as little to as large as the DC/OS advance YAML configuration below.
```

Comment:
This sentence is hard to understand. Please rephrase it. I can't suggest the text as I'm not sure of the meaning. 
"This file can have as little to as large as the DC/OS advance YAML configuration below."